spec/unit/pops/types/iterable_spec.rb

```diff
@@ -0,0 +1,229 @@
+require 'spec_helper'
+require 'puppet/pops'
+
+module Puppet::Pops::Types
+describe 'The iterable support' do
+
+  [5, (3..10), %w(a b c), 'hello', PIntegerType.new(1, 4), PEnumType.new(%w(yes no)) ].each do |obj|
```

Comment:
suggest formatting each input case on a separate row - makes it easier to spot the last line
`].each do |obj| ` and to identify each case